service-base/src/main/resources/META-INF/native-image/org.eclipse.hono/hono-service-base/native-image.properties

```diff
@@ -14,4 +14,4 @@ Args = -H:ResourceConfigurationResources=${.}/resources-config.json \
   --initialize-at-run-time=io.netty.internal.tcnative.SSL \
   -Dvertx.json.base64=legacy \
   -Djdk.tls.ephemeralDHKeySize=matched \
-  -Djdk.tls.rejectClientInitiatedRenegotiation=true
+  -Djdk.tls.rejectClientInitiatedRenegotiation=true \
```

Comment:
IMHO you can revert all changes to this file ...

Reply:
Yes, my mistake being too hasty to push changes.